src/components/ha-date-range-picker.ts

```diff
@@ -413,7 +456,7 @@ export class HaDateRangePicker extends LitElement {
         }
       }
 
-      @media only screen and (max-width: 500px) {
+      @media only screen and (max-width: 560px) {
```

Comment:
Only had troubles on Resolutions between 500 and 560px.
I added 560 here, as min-width:inherit shrinks the date to fit ..

Reply:
@wendevlin: instead hiding it on mobile: this change shrinks the date to fit. If I missed something important here: for sure hiding would be also possible in the first step if really necessary ...

Reply:
The arrows still take too much space on small phones, so I would like to hide it for now and improve the whole input mask with another PR.